nilearn/plotting/find_cuts.py

```diff
@@ -11,11 +11,15 @@
 
 # Local imports
 from .._utils.ndimage import largest_connected_component
-from .._utils import new_img_like
+from .._utils import new_img_like, check_niimg
 from .._utils.extmath import fast_abs_percentile
 from .._utils.numpy_conversions import as_ndarray
 from ..image.resampling import get_mask_bounds, coord_transform
-from ..image.image import _smooth_array
+from ..image.image import _smooth_array, iter_img
+from ..plotting import find_xyz_cut_coords
+import numpy as np
+from scipy import ndimage
```

Comment:
Both the scipy.ndimage and numpy imports are not needed since they are already imported above.